doc/getting-started/installation-instructions/server.md

```diff
@@ -12,43 +12,59 @@ When installing the OpenQuake Engine with the universal installer the `local_set
 
 For example if you clone the repository in the folder `/opt/openquake/src/oq-engine/` you must place the file in `/opt/openquake/src/oq-engine/openquake/server`
 
-Create a `local_settings.py` and add:
-```python
-LOCKDOWN = True
-```
+If you only download install.py file and run the installation, the `local_settings.py` file must be located in `/opt/openquake/venv/lib/python3.11/site-packages/openquake/server` (replacing python3.11 with the actual python version)
+
+Copy `openquake/server/local_settings.py.server` to `openquake/server/local_settings.py`.
+
+#### Configure the STATIC_ROOT Folder
+
+STATIC_ROOT is the full, absolute path to your static files folder.
+Please remember to create the folder /var/www and set the ownership to user openquake.
 
-Upgrade the database to host users and sessions:
 ```console
-$ cd /opt/openquake/src/oq-engine/openquake/server
-$ sudo -u openquake oq webui migrate
+sudo mkdir /var/www
+sudo chown -R openquake /var/www/
 ```
 
-Add a new local superuser:
+#### Configure the directory to store the server user access log
+
+By default, user access information is logged through the standard Django logger.
+In order to write such information to a file, for instance to be digested by Fail2Ban, the variable `WEBUI_ACCESS_LOG_DIR` is specified in `local_settings.py`.
+
+In that case the file `webui-access.log` will be created inside the specified directory.
+Please note that the directory must be created if it does not exist yet.
+Furthermore, the user `openquake` must own that directory.
+
 ```console
-$ cd /opt/openquake/src/oq-engine/openquake/server
-$ sudo -u openquake oq webui createsuperuser
+sudo mkdir /var/log/oq-engine
+sudo chown -R openquake /var/log/oq-engine
 ```
 
-Setup static files in Django
+Upgrade the database to host users and sessions:
 
-Open the file `local_settings.py` and add:
-```python
-# Static Folder
-STATIC_ROOT = '/var/www/webui'
+```console
+cd /opt/openquake/src/oq-engine/openquake/server
+sudo -u openquake oq webui migrate
 ```
-STATIC_ROOT is the full, absolute path to your static files folder.
-Then issue the commands:
+Add a new local superuser:
+
+```console
+cd /opt/openquake/src/oq-engine/openquake/server
+sudo -u openquake oq webui createsuperuser
+```
+To setup static files in Django issue the commands:
```

Comment:
To setup static files in Django, issue the following commands, making sure to refer to the actual folder where the engine was installed (see above):